cmd/build.go

```diff
@@ -183,6 +183,10 @@ func subcommandBuild(targetOS string, packagingTask packaging.Task) {
 		cleanBuildOutputsDir(targetOS)
 		buildFlutterBundle(targetOS)
 	}
+	if !buildDocker && !packagingTask.Supported() {
+		log.Infof("Switching to docker packaging")
+		buildDocker = true
```

Comment:
Please don't automatically switch to docker.
If a user is missing certain tools, we can tell the user to install those tools, or to run `hover` with `--docker`. But I really dislike changing behavior of hover based on something that is an error (missing tools). We should do what the user asks, or give them an error.